detection_rules/devtools.py

```diff
@@ -1044,6 +1046,74 @@ def delta_stats(rule_map) -> List[dict]:
     return changed_stats, new_stats, dep_stats
 
 
+@diff_group.command('packages')
```

Comment:
This has overlap with the `package-stats` and `endpoint-by-attack` commands, so they should probably be under the same click group